### DISCUSSION TITLE
Should medusa.customers.listOrders() return all orders?

### DISCUSSION THREAD

**Discussion by ensidialabs** [medusajs/medusa#3823] [Q&A] [answered]
Hello,
Now all orders are being returned (all order which in db) by medusa.customers.listOrders(), is this how it should be? or should it return only for the buyer who is currently logged in?
```
import Medusa from "@medusajs/medusa-js"
const medusa = new Medusa({ baseUrl: MEDUSA_BACKEND_URL, maxRetries: 3 })
// must be previously logged
medusa.customers.listOrders()
.then(({ orders, limit, offset, count }) => {
  console.log(orders);
});
```

**Comment by olivermrbl** [comment, marked as answer]
`GET /store/customers/me/orders` will only return orders placed by the authenticated customer.

**Reply by ensidialabs** [reply]
was my error, find it. Thank you.
